src/__test__/reducers/signupreducer.test.js

```diff
@@ -0,0 +1,60 @@
+import { isTSAnyKeyword, exportAllDeclaration } from '@babel/types';
```

Comment:
kindly remove this if it's not being used

Reply:
@NicholusMuwonge this has been worked on